datasets/kws20.py

```diff
@@ -416,7 +410,7 @@ def __gen_datasets(self):
             torch.save(mfcc_dataset, os.path.join(self.processed_folder, self.data_file))
 
         print('Dataset created!')
-        print('Training: %d,  Validation: %d, Test: %d' % (traincount, validatecount, testcount))
+        print('Training/Validation: %d,  Test: %d' % (traincount, testcount))
```

Comment:
Instead of "Training/Validation" "Training+Validation" can be more understandable. Same applies in the whole document.

Reply:
Good idea, will do this ASAP.